src/EIPTestsFiller/StateTests/stEOF/stEIP3540/EOF1_ExecutionFiller.yml

```diff
@@ -28,7 +27,7 @@ EOF1_Execution:
       balance: 0
       # Basic valid EOF code
       # code section: ':yul { sstore(0, 1) stop() }'
-      code: ':raw 0xef000101000402000100060300000000000002600160005500'
+      code: ':raw 0xef000101000402000100060400000000000002600160005500'
```

Comment:
This file not updated yet?

Reply:
Good catch, this file was not updated to change first code section output to non-returning function. It is interesting because, evmone is filling the test correctly... this means... it is executing this invalid code in pre-state. Currently there is a pre-state EOF code validation when running the tests, but there is not validation when filling the tests. I have updated the file and refilled the tests. Also submitted [this PR in the evmone repo](https://github.com/ethereum/evmone/pull/696) in order to validate eof code in pre-state when filling tests.